src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -50,6 +50,7 @@
     <ExpectedDependency Include="Humanizer.Core"/>
     <ExpectedDependency Include="ICSharpCode.Decompiler"/>
     <ExpectedDependency Include="Microsoft.DiaSymReader"/>
+    <ExpectedDependency Include="Microsoft.CodeAnalysis.FlowAnalysis.Utilities"/>
```

Comment:
Why do we insert this assembly with Roslyn, shouldn't it be done from analyzer repo directly? Is this an intermediate step until we get the infrastructure in place?

Reply:
Primarily, yes. Also note that we do not publish this NuGet package on nuget.org yet as it is not still ready for public consumption.